test/weets.test.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable */
+const request = require('supertest');
+const api = require('../config/axios');
+
+const app = require('../app');
+const db = require('../app/models');
+const { buildUser } = require('./factory/users');
+
+jest.mock('../config/axios');
+
+let user = '';
+
+beforeEach(async () => {
+  await db.User.destroy({ truncate: { cascade: true } });
+  await db.Weet.destroy({ truncate: { cascade: true } });
```

Comment:
Why are you doing this truncate?

Reply:
The idea was to clean the tables before each test but I just realized that it was not necessary.